components/brave_shields/browser/domain_block_navigation_throttle.cc

```diff
@@ -87,8 +87,12 @@ DomainBlockNavigationThrottle::WillStartRequest() {
   if (!ad_block_service_->IsInitialized())
     return content::NavigationThrottle::PROCEED;
 
-  // Don't bother checking non-HTTP(S) pages
+  // Don't block subframes
   content::NavigationHandle* handle = navigation_handle();
+  if (!handle->IsInMainFrame())
```

Comment:
can we add a test for this?

Reply:
Working on this now; will post in a new PR